controller/api/public/grpc_server_test.go

```diff
@@ -10,9 +10,8 @@ import (
 
 	"github.com/golang/protobuf/proto"
 	"github.com/golang/protobuf/ptypes/duration"
-	"github.com/linkerd/linkerd2/controller/api/destination"
```

Comment:
is this the change intended to `update the public API package to stop importing the destination API package, by introducing a shared discovery API package on which they can both depend`?

Reply:
That's right. My overall goal was to stop having this test file call `destination.InitFakeDiscoveryServer`, and I did that by way of introducing `discovery.MockDiscoveryClient` instead.